### PR TITLE
Commerce 7923 Import from CSV - Use SKU code in import file

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-service/service.xml [hunk 1]
```diff
@@ -674,6 +674,9 @@
 		<finder name="CPInstanceUuid" return-type="Collection">
 			<finder-column name="CPInstanceUuid" />
 		</finder>
+		<finder name="Sku" return-type="Collection">
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-service/service.xml:677]: This should be C_S (companyId, sku)

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/importer/type/util/CommerceOrderImporterTypeUtil.java [hunk 2]
```diff
@@ -224,6 +231,27 @@
...
+	public static CSVFormat getCSVFormat(
+		CommerceOrderImporterTypeConfiguration
+			commerceOrderImporterTypeConfiguration) {
+
+		CSVFormat csvFormat = CSVFormat.DEFAULT;
+
+		String csvSeparator =
+			commerceOrderImporterTypeConfiguration.csvSeparator();
+
+		if (Objects.equals(csvSeparator, StringPool.SEMICOLON)) {
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/importer/type/util/CommerceOrderImporterTypeUtil.java:243]: Use StringPool.SEMICOLON.equals(csvSeparator) instead

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@luca-pellizzon Take a look at the changes requested.
Thanks!

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Pull request closed without merging**